content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +200,80 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path for patch release (bug fixes)
+
+Within a channel each patch release should be directly upgradable to the `head channel` from the latest version shipped 
+in the minor channel behind. Use skips or skipRange to provide this behaviour. (i.e. using the skipRange in the 
+`stable-3.6` patch for `3.5.<latest>: >= 3.5.<first z of the minor channel>. < 3.6.<new pacth release>`)
+
+#### Example
+
+Let's imagine that you will publish the Operator bundle `3.6.30` (patch release under the channel `stable-3.6`) and
+that you have published so far:
+
+- Operator bundles versions from `3.5.0` to `3.5.25` under the `stable-3.5` channel
+- Operator bundles versions from `3.6.0` to `3.6.29` under the `stable-3.6` channel
+
+Then, in this case, your newer patch release `3.6.30` would be configured with `skipRange: >=3.5.25 < 3.6.30` in
+order to only supports upgrading to the newest `3.6.z` from a pretty recent `3.5.z`
```

Comment:
They shouldn't necessarily use 3.5.25, they should use the oldest 3.5.z version that they can support upgrades from.

they can use 3.5.25 if, for some reason, they don't think upgrades from something older like 3.5.24 (or more likely, 3.5.0) to 3.6.latest will work.  But it the ideal situation is that they do allow upgrades from 3.5.0 to 3.6.latest (i.e. `skipRange: >=3.5.0 < 3.6.30`).  The ability to set it to 3.5.25 (or 3.5.17, or something else) instead of 3.5.0 is a mechanism to constrain the support matrix *when necessary*, it's not the default recommendation.